CONTRIBUTING.md

```diff
@@ -185,22 +185,19 @@ Deep Learning Inference Engine (IE) supports models in the Intermediate Represen
 
 ## Demo
 
-A demo shows the main idea of how to infer a model using IE. If your model solves one of the tasks supported by the Open Model Zoo, try to find an appropriate option from [demos](demos/README.md) or [samples](https://docs.openvino.ai/latest/_docs_IE_DG_Samples_Overview.html). Otherwise, you must provide your own demo (C++ or Python).
+A demo shows the main idea of how to infer a model using OpenVINO™. If your model solves one of the tasks supported by the Open Model Zoo, try to find an appropriate option from [demos](demos/README.md) or [samples](https://docs.openvino.ai/latest/_docs_IE_DG_Samples_Overview.html). Otherwise, you must provide your own demo (C++ or Python).
 
 The demo's name should end with `_demo` suffix to follow the convention of the project.
 
-Demos are required to support the following keys:
+Demos are required to support the following args:
 
- -  `-i "<input>"`: Required. An input to process. The input can usually be a single image, a folder of images or anything that OpenCV's `VideoCapture` can process.
- -  `-m "<path>"`: Required. Path to an .xml file with a trained model. If the demo uses several models at the same time, use other keys prefixed with `-m_`.
- -  `-d "<device>"`: Optional. Specifies a target device to infer on. CPU, GPU, HDDL or MYRIAD is acceptable. Default must be CPU. If the demo uses several models at the same time, use keys prefixed with `d_` (just like keys `m_*` above) to specify device for each model.
- -  `-no_show`: Optional. Do not visualize inference results.
+* `-h, --help`        show this help message and exit
+* `-m <MODEL FILE>`   path to an .xml file with a trained model. If the demo uses several models at the same time, use format `--m?*`.
```

Comment:
```suggestion
* `-m <MODEL FILE>`   path to an .xml file with a trained model. If the demo uses several models at the same time, use format `-m?*`.
```

Reply:
`-m?*` is at least 2 symbol flag, so --

Reply:
Though in fact we will support both

Reply:
I don't think I see the point.
Your note
> better to have `-m` as a general rule

is satisficed by `-m <MODEL FILE>`   path to an .xml file with a trained model.`

The note
> comment in description, like "some demos might use extended syntax, like -m_det"

is implemented by `If the demo uses several models at the same time, use format --m?*`

Reply:
point is that this "regular expression" like `?*` might be not obvious. While explaining that "might use extended syntax, like `-m_det` seems to be more clear

Reply:
Updated. I also removed `_`. That allows to mimic `-nstreams` arg

Reply:
The point of my suggestion was to left only one `-` because the documentation describes the flags for both c++ and python demos. There are no `--m*?` in c++ demos
According to our test, `--mdet` is not the common view of the flag. In almost demos we use `-m_det`, `-m_ag` etc.

Reply:
I think we should change this style

Reply:
Okay